### ISSUE TITLE
Brackets Sprint-43 Conflict with ReadAsText:

### ISSUE THREAD

**Issue by AyoungDukie** [MiguelCastillo/Brackets-Themes#105]
I'm probably titling this wrong, but what I am trying to explain is that with the most recent update brackets develops internal errors for me when the Themes extension is enabled. Mainly (from the browser Developer Tools).

![capture4](https://cloud.githubusercontent.com/assets/5449443/4340800/0798ea4a-4032-11e4-8900-e297b6abd427.PNG)

I'm not sure if this is a brackets issue, (I don't think it is theme-dependent, as it also occurred above with the main theme), and functionally everything still seems to work, just more of a concern than anything.

Thank you though, the extension is wonderful otherwise, I am glad to have it!

**Comment by jtflynnz** [comment]
Sounds great, good luck!

**Comment by MiguelCastillo** [comment]
@jtflynnz i never got to digging into this issue because I stopped seeing at some point. please let me know if you still this issue.

**Comment by jtflynnz** [comment]
No, this no longer seems to pop up with the latest version of Brackets (1.2), so it must have been something with the version at the time. Thanks for checking back though!
